src/collection/components/modals/CutFragmentModal.tsx

```diff
@@ -150,22 +150,19 @@ const CutFragmentModal: FunctionComponent<CutFragmentModalProps> = ({
 		<Modal
 			isOpen={isOpen}
 			title={t('collection/components/modals/cut-fragment-modal___knip-fragment')}
-			size="medium"
+			size="auto"
 			onClose={onClose}
 			scrollable
 			className="m-cut-fragment-modal"
 		>
 			<ModalBody>
-				<FlowPlayerWrapper
-					item={itemMetaData}
-					external_id={itemMetaData.external_id}
-					duration={itemMetaData.duration}
-					title={itemMetaData.title}
-					cuePointsLabel={{
-						start: fragmentStartTime,
-						end: fragmentEndTime,
-					}}
+				<ItemVideoDescription
+					itemMetaData={itemMetaData}
+					showTitle
+					showDescription={false}
 					canPlay={isOpen}
+					cuePoints={{ start, end }}
```

Comment:
Since the old code used these values, i'm assuming the changed code should use the same values.
```suggestion
					cuePoints={{ 
						start: fragmentStartTime,
						end: fragmentEndTime,
					}}
```